src/lib/core/tests/TestCHIPTLV.cpp

```diff
@@ -485,11 +639,11 @@ void ReadEncoding1(nlTestSuite * inSuite, TLVReader & reader)
 
         TestNext<TLVReader>(inSuite, reader2);
 
-        TestGet<TLVReader, bool>(inSuite, reader2, kTLVType_Boolean, ProfileTag(TestProfile_1, 2), true);
```

Comment:
Why all the changes away from using TestGet?

Reply:
All `Test*()` inline functions in this file have the same issue: they are functions, and, therefore, the assertion macros in those functions get expended before the functions are called/inlined. As the result, shown locations of assertion failures have nothing to do with the locations of actual problems.

The reason I have not completely removed `TestGet` was that for some reason `TEST_GET` macro had a problem with float/double types -- still need to figure out what is wrong.

Reply:
> As the result, shown locations of assertion failures have nothing to do with the locations of actual problems.

Ah, right.  Would have been handy to put that in the commit message.  ;)